src/containers/WelcomePage/components/Revisions.tsx

```diff
@@ -153,6 +190,18 @@ const Revisions = ({ userData, ndlaId }: Props) => {
             infoText={t('welcomePage.revisionInfo')}
           />
           <ControlWrapperDashboard>
+            <Tooltip tooltip={t('welcomePage.primaryConnection')}>
+              <SwitchWrapper>
+                <Switch
+                  checked={checked}
+                  onChange={() => setChecked(!checked)}
+                  label={''}
+                  id={'filter-primary-connection-switch'}
```

Comment:
Skal label være tom?
```suggestion
                  label=""
                  id="filter-primary-connection-switch"
```

Reply:
Vi har egentlig ikke plass til label i den komponenten.. Det er ikke ideellt UU-messig, men la på aria-label istedet. En alternativ løsning som kanskje er bedre er å legge inn mulighet for en hidden label i Switch-komponenten i frontend-packages. 

Kanskje @Jonas-C har noen meninger her?

Reply:
Tja, jeg har alltid en mening. Hva med å legge denne under de to knappene, og forkorte label til `Vis kun favorittfag`? Tooltip kan forbli like lang.

Reply:
Hmm ja, å legge under er en mulighet, men for at labelen skal gi mening må den hvertfall være noe sånt som: `Vis kun ressurser med primærkobling`, og det tar jo en del plass i bredden ..